easybuild/easyconfigs/x/XCfun/XCfun-20180122-intel-2017b-Python-2.7.14.eb

```diff
@@ -19,7 +19,7 @@ patches = ['%(name)s-%(version)s-shared_library-intel.patch']
 checksums = [
     # acd11bfbdcc2d2fb9c100111545bd479e6e2c452.tar.gz
     '1dd1bac10fd12682ac0b65c51fd1fd5928104d1210165ef7d9e433ec99090e4e',
-    # XCfun-20180122-shared_library-intel.patch
+    # XCfun-20180122.tar.gz
```

Comment:
@smoors You fixed the wrong comment... ;)

Reply:
oops haast en spoed en zo :)